demo/app.js

```diff
@@ -463,7 +465,21 @@ const Demo = React.createClass({
 
     setTimeout(() => {
       this.setState({
-        lineChartData
+        lineChartData,
+        donutChartData: [
+          {
+            name: 'Data Point 1',
+            value: 50
+          },
+          {
+            name: 'Data Point 2',
+            value: 80
+          },
+          {
+            name: 'Data Point 3',
+            value: 200
+          }
+        ]
```

Comment:
I know the setTImeout was already in here, but I'm assuming it is just mimicking an async api response for the demo?